File: src/python/greedy01.py
### 그리디 알고리즘(탐욕법) ###
# 거스름 돈
n = 1260
count = 0
array = [500, 100, 50, 10]

for coin in array:
  count = count + n // coin
  n = n % coin

print(count)

File: src/ndb_greedy/greedy01.java
package ndb_greedy;

public class greedy01 {
	public static void main(String[] args) {
		int n = 1260;
		int count = 0;
		int[] coinType = {500, 100, 50, 10};
		
		for(int i=0; i<coinType.length; i++) {
			count += n / coinType[i];
			n %= coinType[i];
		}
		System.out.println(count);
	}
}